### PR TITLE
DDPM Tutorials

### PR DESCRIPTION
**Pull request by ericspod**
Address some of #1769.

### Description
These are the 2D ~~and 3D~~ DDPM tutorials. I have run them again to verify they work. ~~The 2D ones are fine but the 3D tutorial keeps having mode collapse or some other issue resulting in bad results. Work is ongoing on this one.~~ I have left the 3D tutorial off since it's not training stably and needs some revamp to be a proper and useful tutorial for users.

### Checks

- [x] Avoid including large-size files in the PR.
- [x] Clean up long text outputs from code cells in the notebook.
- [x] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [x] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [x] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

**Comment by ericspod** [comment]
I'm still trying to get good results for the 3D example so not ready for review yet.

File: generation/2d_ddpm/2d_ddpm_inpainting.ipynb [hunk 1]
```diff
@@ -0,0 +1,757 @@
+{
```
Comment (KumoLiu) [inline generation/2d_ddpm/2d_ddpm_inpainting.ipynb:1]: <p>I guess here you mean from the generative repo? May need mention it here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1814/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (KumoLiu) [inline generation/2d_ddpm/2d_ddpm_inpainting.ipynb:1]: <p>Misadded here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1814/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

File: generation/2d_ddpm/2d_ddpm_tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,709 @@
+{
```
Comment (KumoLiu) [inline generation/2d_ddpm/2d_ddpm_tutorial.ipynb:1]: <p>The same as before here, perhaps need add generative repo link here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1814/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Review by KumoLiu — APPROVED** [review]
Thanks for the PR! Overall looks good to me. Minor comments inline.
And could you please also update the README file here: https://github.com/Project-MONAI/tutorials/blob/main/generation/README.md? Thanks!

**Review by virginiafdez — APPROVED** [review]
Besides existing comments, they look fine by me.

File: generation/2d_ddpm/2d_ddpm_inpainting.ipynb (hunk 1 above)
Comment (ericspod) [reply generation/2d_ddpm/2d_ddpm_inpainting.ipynb:1]: <p>This is something that should be changed to reference the distributed training tutorial here in the mail repo once this is ported from GenerativeModels</p><p> </p>

**Comment by KumoLiu** [comment]
Hi @ericspod, I just disable the auto-merge since the premerge-notebook is not a required one to test. Want to double-confirm. Perhaps we need set it to required test.

**Comment by ericspod** [comment]
It should be required I'd say, unless there was some reason why notebooks would fail but still be mergable.

**Comment by KumoLiu** [comment]
> It should be required I'd say, unless there was some reason why notebooks would fail but still be mergable.

Let me try whether I have permission to set.

**Pull request merged**